lib/matplotlib/dates.py

```diff
@@ -201,6 +220,62 @@ def _get_rc_timezone():
     MO, TU, WE, TH, FR, SA, SU)
 WEEKDAYS = (MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY)
 
+# default epoch: passed to np.datetime64...
+_epoch = None
+
+
+def _reset_epoch_test_example():
+    """Reset the Matplotlib date epoch so it can be set again."""
```

Comment:
Let's be very clear about it:
```suggestion
    """
    Reset the Matplotlib date epoch so it can be set again.
    
    This function should only be used in tests.
    """
```